include/core/parameters.h

```diff
@@ -248,6 +220,38 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Fluid - Class for fluid definition
+   */
+  class Fluid
+  {
+  public:
+    Fluid()
+    {}
+
+    void
+    declare_parameters(ParameterHandler &prm, unsigned int id);
+    void
+    parse_parameters(ParameterHandler &prm, unsigned int id);
+
+    // Kinematic viscosity (nu = mu/rho) in units of L^2/s
```

Comment:
```suggestion
    // Kinematic viscosity (nu = mu/rho) in units of m^2/s
```

Reply:
I will do a commit and apply all of your suggestions Shahab :)